repository/kafka/docs/v0.2/install.md

```diff
@@ -0,0 +1,42 @@
+# Installing the KUDO Kafka Operator
+
+Requirements:
+
+- Install the [KUDO controller](https://kudo.dev/docs/getting-started/)
+- Install the [KUDO CLI](https://kudo.dev/docs/cli/)
+
+
+## Installing the Operator
+
+#### Install Zookeeper 
+```
+kubectl kudo install zookeeper --instance=zk
+```
+
+#### Install Kafka 
+```
+kubectl kudo install kafka --instance=kafka
```

Comment:
Hm, based on the `ZOOKEEPER_PATH` instructions above, would this work? 
If you install zookeeper under `/zk`, wouldn't installing `kafka` fail because it thinks the `ZOOKEEPER_PATH=/kafka`?

Reply:
I added an explanation around `ZOOKEEPER_PATH` its not the zookeeper connection uri but zk node that hosts the data.